test/command/suite/sharding/logical_count/log/select.test

```diff
@@ -5,6 +5,11 @@ plugin_register sharding
 table_create Logs_20150709 TABLE_NO_KEY
 column_create Logs_20150709 timestamp COLUMN_SCALAR Time
 
+load --table Logs_20150709
+[
+{"timestamp": "2015-02-05 13:49:00"}
+]
```

Comment:
I have added this because the expected log `#|d| [logical_count][select] <Logs_20150709>: no range index` is not written if `Logs_20150709` is empty.

Currently, an empty table is skipped before logging that log.
https://github.com/groonga/groonga/pull/1341/files#diff-d3fb57a047ca92c9fc19d900aa1a906a31bc3519eb7c26927516b229705c27a4R290